portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,30 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private ArrayList<String> commentList = new ArrayList<String>();
```

Comment:
would recommend making this a generic `List`. That way you can just change the `ArrayList` to, for example, a `LinkedList` and nothing else in your code would need to change

Reply:
Makes sense! Let me fix that